### PR TITLE
Good contract found and working

### PR DESCRIPTION
**Pull request by AGradowski**

### CODE REVIEW

File: auth/Auth/appsettings.json [hunk 1]
```diff
@@ -7,6 +7,6 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "AuthPostgresConnection": "Server=localhost;Port=5432;Database=auth;Username=someuser;Password=somePassword"
+    "AuthPostgresConnection": "Server=localhost;Port=5011;Database=auth;Username=someuser;Password=somePassword"
```
Comment (JanuszJakubiec) [inline auth/Auth/appsettings.json:10]: I don't understand why have you commited this, but ok

File: contracts/contracts/Rental.sol [hunk 2]
```diff
@@ -93,7 +96,9 @@ contract Rental {
 
         rentalHistory.push(newRecord);
 
-        rentalMappingEnd[record.rentalID] = rentalHistory.length -1;
+          
+
+        rentalMappingEnd[record.rentalID] = rentalHistory.length ;
```
Comment (JanuszJakubiec) [inline contracts/contracts/Rental.sol:101]:
```suggestion
        rentalMappingEnd[record.rentalID] = rentalHistory.length;
```

**Review by JanuszJakubiec — DISMISSED** [review]

File: auth/Auth/appsettings.json (hunk 1 above)
Comment (AGradowski) [reply auth/Auth/appsettings.json:10]: Because this is the working configuration (on my PC, true, but my PC is the only one runnig it at the moment)

**Review by JanuszJakubiec — APPROVED** [review]
looks good

**Pull request merged**